dockerfiles/vcpkg/build.sh

```diff
@@ -51,9 +51,14 @@ function doBuild() {
     elif [ "$1" == "amazonlinux" ]; then
         CMAKE_OPTIONS_EXTRA="-DVCPKG_TARGET_TRIPLET=x64-amazonlinux-dynamic"
     fi
-    docker run --rm --mount source="$(pwd)",target=/hpcc-dev/HPCC-Platform,type=bind,consistency=cached hpccsystems/platform-build-$1:$VCPKG_REF \
-        "cmake -S /hpcc-dev/HPCC-Platform -B /hpcc-dev/HPCC-Platform/build-$1 ${CMAKE_OPTIONS} ${CMAKE_OPTIONS_EXTRA} && \
-        cmake --build /hpcc-dev/HPCC-Platform/build-$1 --target package --parallel $(nproc)"
+    mkdir -p $HOME/.ccache
+    docker run --rm \
+        --mount source="$(pwd)",target=/hpcc-dev/HPCC-Platform,type=bind,consistency=cached \
+        --mount source="$HOME/.ccache",target=/root/.ccache,type=bind,consistency=cached \
+        hpccsystems/platform-build-$1:$VCPKG_REF \
+        "cmake -S /hpcc-dev/HPCC-Platform -B /hpcc-dev/HPCC-Platform/build-$1 ${CMAKE_OPTIONS} ${CMAKE_OPTIONS_EXTRA} -DCMAKE_C_COMPILER_LAUNCHER=ccache -DCMAKE_CXX_COMPILER_LAUNCHER=ccache && \
+        cmake --build /hpcc-dev/HPCC-Platform/build-$1 --target install --parallel $(nproc) && \
+        /etc/init.d/hpcc-init start"
```

Comment:
Only used for local testing - should revert...

Reply:
is the whole of vcpkg/build.sh only used for testing (not part of the workflow build actions) ?

Reply:
Correct - its basically a way to locally test builds for other OSs